CuteLogger/src/AbstractAppender.cpp

```diff
@@ -38,7 +38,7 @@
  * imagine.
  *
  * For the simple non-structured plain text logging (for example, to a plain text file or to the console output) you may
- * like to subclass the AbstractStringAppender instead of AbstractAppender, which will give you a more convinient way to
+ * like to subclass the AbstractStringAppender instead of AbstractAppender, which will give you a more convenient way to
```

Comment:
CuteLogger is not our code; it is imported from another project. I do not want to revise all their code like this.